### PR TITLE
blight: add page

### LINKED ISSUE #8522: page request: blight
### Command description

A command line utility to change the display brightness. See: https://github.com/gutjuri/blight

Not sure whether or not it is allowed to request a page for a programme that I've written myself, so feel free to delete.

### Command details

_No response_

### OS

- [ ] Android
- [X] Linux
- [ ] macOS
- [ ] sunOS
- [ ] Windows

### PR DESCRIPTION
**Pull request by kruczyna**
- [x] The page(s) are in the correct platform directories: `linux`.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

Closes #8522

### CODE REVIEW

**Comment by kbdharun** [comment]
@gutjuri Can you check this PR? Thanks in advance.

> I am confused as the linked issue links to [Clight](https://github.com/FedeDP/Clight) and this PR is for `blight`. Is this program a fork of `clight`.

File: pages/linux/blight.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- Show current display brightness:
+
+`blight show`
+
+- Show maximum display brightness:
+
+`blight max`
+
+- Increase display brightness with %:
```
Comment (Nico385412) [inline pages/linux/blight.md:18]:
```suggestion
- Increase display brightness in %:
```

what do you think ? is it better ? 🤔

File: pages/linux/blight.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# blight
+
+> Utility for changing the display brightness.
+> More information: <https://github.com/gutjuri/blight>.
+
+- Set display brightness:
+
+`blight set`
```
Comment (gutjuri) [inline pages/linux/blight.md:8]:
```suggestion
`blight set {{50}} -r`
```

File: pages/linux/blight.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# blight
+
+> Utility for changing the display brightness.
+> More information: <https://github.com/gutjuri/blight>.
+
+- Set display brightness:
```
Comment (gutjuri) [inline pages/linux/blight.md:6]:
```suggestion
- Set display brightness to 50%:
```

**Review by gutjuri — CHANGES_REQUESTED** [review]

**Comment by gutjuri** [comment]
> @gutjuri Can you check this PR? Thanks in advance.

Yes, I left a review.

> > I am confused as the linked issue links to [Clight](https://github.com/FedeDP/Clight) and this PR is for `blight`. Is this program a fork of `clight`.

The correct issue is #8522. blight is not a fork of clight; these are completely different issues.

**Comment by kbdharun** [comment]
> > @gutjuri Can you check this PR? Thanks in advance.
> 
> Yes, I left a review.

Thanks ❤️ 

> > > I am confused as the linked issue links to [Clight](https://github.com/FedeDP/Clight) and this PR is for `blight`. Is this program a fork of `clight`.
>
> The correct issue is #8522. blight is not a fork of clight; these are completely different issues.

Thanks for the information 👍🏽 , I will update the mentioned issue in the PR description.

**Review by gutjuri — APPROVED** [review]

File: pages/linux/blight.md (hunk 3 above)
Comment (kruczyna) [reply pages/linux/blight.md]: Applied changes in [blight: adjust to change requests](https://github.com/tldr-pages/tldr/pull/8633/commits/3f7059197d72071c9d57bd58b73cc50760504011) :)

File: pages/linux/blight.md (hunk 1 above)
Comment (kruczyna) [reply pages/linux/blight.md]: Applied changes in [blight: adjust to change requests](https://github.com/tldr-pages/tldr/pull/8633/commits/3f7059197d72071c9d57bd58b73cc50760504011) 🌻

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/linux/blight.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`blight show`
+
+- Show maximum display brightness:
+
+`blight max`
+
+- Increase display brightness in %:
+
+`blight inc {{your-number}} -r`
```
Comment (CleanMachine1) [inline pages/linux/blight.md:20]:
```suggestion
`blight inc {{number}} -r`
```

File: pages/linux/blight.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Decrease display brightness with internal units:
+
+`blight dec {{your-number}}`
```
Comment (CleanMachine1) [inline pages/linux/blight.md:24]:
```suggestion
`blight dec {{number}}`
```

**Review by CleanMachine1 — APPROVED** [review]

File: pages/linux/blight.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+- Set display brightness to 50%:
+
+`blight set {{50}} -r`
+
+- Show current display brightness:
+
+`blight show`
+
+- Show maximum display brightness:
```
Comment (navarroaxel) [inline pages/linux/blight.md:14]: This description is not clear to me, prints the value or applies the max brightness to the screen?

Comment (gutjuri) [reply pages/linux/blight.md:14]: It prints the value.

Comment (kruczyna) [reply pages/linux/blight.md]: Changed the command to "Print maximum display value" in [blight: fix arguments and fix command desc](https://github.com/tldr-pages/tldr/pull/8633/commits/5149693ccbee60351349bd00501c0f4d5f6d07d1)

**Pull request merged**